charts/seed-bootstrap/dashboards/client-go.json

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
Can you add these dashboards to the control plane monitoring as well?
We have controller-runtime components there (e.g. grm) which operators want to monitor and inspect as well.

There is already a copy of the resource manager dashboard which could be replaced now with the generic dashboards: https://github.com/gardener/gardener/blob/aebd83e58f605a8bcf47512879b45375cbd3582f/charts/seed-monitoring/charts/grafana/dashboards/operators/resource-manager.json

Reply:
Addressed in https://github.com/gardener/gardener/pull/7180/commits/956dfa151bf85a08002b19de3021a0ffab66b93b